bson/src/test/unit/util/JsonPoweredTestHelper.java

```diff
@@ -42,9 +44,14 @@ public static BsonDocument getTestDocument(final String resourcePath) throws IOE
     }
 
     public static List<File> getTestFiles(final String resourcePath) throws URISyntaxException {
-        List<File> files = new ArrayList<File>();
-        addFilesFromDirectory(new File(JsonPoweredTestHelper.class.getResource(resourcePath).toURI()), files);
-        return files;
+        URL resource = JsonPoweredTestHelper.class.getResource(resourcePath);
```

Comment:
I guess this is needed because currently there are no unified change stream tests?  

But I worry that by not throwing an exception in this case we might inadvertently hide a mistake in the resourcePath passed by the caller.

Reply:
Yes, that may be a problem. I can add a dummy file with no tests and remove this check. Would such solution be acceptable?

Reply:
Added `unified-test-format/change-streams/change-streams.json` without tests.